rs/tests/node/BUILD.bazel

```diff
@@ -9,5 +9,5 @@ system_test_nns(
     proc_macro_deps = MACRO_DEPENDENCIES,
     target_compatible_with = ["@platforms//os:linux"],  # requires libssh that does not build on Mac OS
     runtime_deps = GUESTOS_RUNTIME_DEPS,
-    deps = DEPENDENCIES + ["//rs/tests"],
+    deps = DEPENDENCIES,
```

Comment:
It would be great if you could also not use [`DEPENDENCIES`](https://sourcegraph.com/github.com/dfinity/ic/-/blob/rs/tests/common.bzl?L7) since it will probably pull in many deps you don't need.

Reply:
So ideally your `deps` look like the `deps` in https://sourcegraph.com/github.com/dfinity/ic/-/blob/rs/tests/consensus/BUILD.bazel. Nice and small.

Reply:
Done.